packages/firebase_auth/firebase_auth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.1.0
+
+- **FEAT**: linkWithPopup() now possible on web.
```

Comment:
Please remove all `CHANGELOG.md` edits because this is handled by [Melos](https://github.com/invertase/melos) when we do a release, the tool we use for managing the repo.